README.md

```diff
@@ -47,6 +47,9 @@ Unsure what to contribute? Check out the `good first issue` [tagged Github issue
   - [How to MAKE CHROMATICS (Friday Night Funkin) - bbpanzu](https://www.youtube.com/watch?v=a7SGu1fNthc)
   - [EASY FNF chromatic guide! (Friday Night Funkin') - StickyBM](https://youtu.be/PlSh_LJwQD0)
   - [How to make FNF Chromatics in Ableton - LongestSoloEver](https://www.youtube.com/watch?v=QCA6-N-pW_0)
+  - [Polyphone Soundfont Editor](https://www.polyphone-soundfonts.com/) - A soundfont editor for those who want to make music quickly using their pitched samples.
+- Soundfont Guides
```

Comment:
Instead of the `-` add three hashtags (###).

Reply:
(this is for line 51 btw)

Reply:
okay, thanks!